server.js

```diff
@@ -50,17 +52,25 @@ app.get('/profile', (req, res, next) => {
 app.post('/addStudent', (req, res, next) => {
   db.save('student', req.body.student)
     .then((data) => {
-      res.end('You saved a new student with the data: ' + JSON.stringify(data, undefined, 2));
+      res.end(JSON.stringify(data));
     });
 });
 
 app.get('/getAllStudents', (req, res, next) => {
   db.loadAll('student')
     .then((data) => {
-      res.end('Here is all of our students: \n' + JSON.stringify(data, undefined, 2));
+      res.end(JSON.stringify(data));
     })
 });
 
+app.get('/getStudent:id', (req, res, next) => {
```

Comment:
I have a feeling we won't be needing this anymore. But we can leave it in for now, as we discussed.